docs/learn/bitesize-invaders.md

```diff
@@ -0,0 +1,309 @@
+---
+id: bitesize-invaders
+title: Invaders Sample
+description: Learn more about game flow, modes, unconventional movement networked, and a shared timer using MLAPI.
+---
+
+The [Invaders Sample Project](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/master/Basic/Invaders) to understand game flow amd modes with Unity MLAPI using Scene Management, unconventional movement networked, and a shared timer between clients in a client-sided predicted way.
+
+## Game Flows
+
+Most Multiplayer games have multiple Networked Scenes, where players can join, communicate, and progress through scenes and maps together. This is a game flow: players join and establish communication together, the server determines the next scene or map, and transitions all clients to the new scene and loads the required map. This ensures players can play together.
+
+The logic and transitions are a specified game flow. To transition in a smooth manner, you need to create game flows and back-end systems that support those flows.
+
+Invaders implements this game-flow by creating different controller classes that handle each Game Mode such as MainMenu, a simple Networked Lobby and InGame modes, and other support classes that help with the transition from one Game Mode to another.
+
+The backbones of the flow/system mentioned above is consisting of 2 main components:
+
+* `SceneTransitionHandler`
+* `SceneState` 
+* Lobby Controller
+
+### SceneTransitionHandler
+
+A `SceneTransitionHandler` with a lightweight state machine allows you to track clients' progress in regards to Scene Loading. It notifies the server when clients finish loading, so that the other listeners are informed, by encapsulating the `SceneSwitchProgress` or creating a wrapper around it that others can subscribe to.
```

Comment:
```suggestion
A `SceneTransitionHandler` with a lightweight state machine allows you to track clients' progress in regards to Scene Loading. It notifies the server when clients finish loading so that the other listeners are informed, by encapsulating the `SceneSwitchProgress` or creating a wrapper around it that others can subscribe to.
```